tests/Unit/Infrastructure/Mail/ErrorHandlingTemplateBasedMailerTest.php

```diff
@@ -0,0 +1,33 @@
+<?php
+
+declare( strict_types = 1 );
+
+namespace WMDE\Fundraising\Frontend\Tests\Unit\Infrastructure\Mail;
+
+use PHPUnit\Framework\TestCase;
+use Psr\Log\LogLevel;
+use WMDE\EmailAddress\EmailAddress;
+use WMDE\Fundraising\Frontend\Infrastructure\Mail\ErrorHandlingTemplateBasedMailer;
+use WMDE\Fundraising\Frontend\Tests\Fixtures\ErrorThrowingTemplateBasedMailer;
+use WMDE\PsrLogTestDoubles\LoggerSpy;
+
+/**
+ * @covers \WMDE\Fundraising\Frontend\Infrastructure\Mail\ErrorHandlingTemplateBasedMailer
+ */
+class ErrorHandlingTemplateBasedMailerTest extends TestCase {
+
+	public function testOnSendMailWithError_logsError(): void {
+		$logger = new LoggerSpy();
+
+		$errorHandlingMailer = new ErrorHandlingTemplateBasedMailer(
+			new ErrorThrowingTemplateBasedMailer(),
+			$logger
+		);
+
+		$errorHandlingMailer->sendMail( new EmailAddress( 'happy@bunny.carrot' ) );
+
+		$this->assertSame( 1, $logger->getLogCalls()->count() );
+		$this->assertSame( [ ErrorThrowingTemplateBasedMailer::ERROR_MESSAGE ], $logger->getLogCalls()->getMessages() );
+		$this->assertSame( LogLevel::ERROR, $logger->getLogCalls()->getFirstCall()->getLevel() );
+	}
+}
```

Comment:
Although `ErrorHandlingTemplateBasedMailer` has 100% coverage, you could also test the happy path with a `TemplateBasedMailerSpy` to document that in the absence of errors the `ErrorHandlingTemplateBasedMailer` just passes on the call.